components/form/index.en-US.md

```diff
@@ -275,3 +275,55 @@ validator(rule, value, callback) => {
   }
 }
 ```
+
+### How you can use `wrappedComponentRef` to get form instance in functional component
```

Comment:
```Why `wrappedComponentRef` get warning with function component?```

Reply:
warning: `Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()`

Reply:
I mean change the title to:

> Why `wrappedComponentRef` get warning with function component?